Objects/Objects/Geometry/Brep.cs

```diff
@@ -306,6 +307,33 @@ internal void OnDeserialized(StreamingContext context)
             f.Brep = this;
       }
     }
+
+    /// <summary>
+    /// Returns a new transformed Brep. Note that `displayMesh` is *NOT* transformed. If you fall back on
+    /// the `displayMesh`, make sure to call `TransformTo` on the mesh object.
+    /// </summary>
+    /// <param name="transform"></param>
+    /// <returns></returns>
+    public Brep TransformTo(Transform transform)
```

Comment:
Should consider redesigning these methods to return bool (or some other way of indicating a successful operation), since anything that implements `ApplyToCurves` can result in invalid geo which will be difficult to troubleshoot later.

Reply:
that's a fair point! could we make an `out bool result` value? i think it's quite friendly for the method itself to return the transformed obj

Reply:
good point - refactor pushed! have a looksie ^^